src/apps/answers/api.py

```diff
@@ -634,3 +636,31 @@ async def answers_existence_check(
     )
 
     return Response[AnswerExistenceResponse](result=AnswerExistenceResponse(exists=is_exist))
+
+
+async def applet_validate_take_now(
+    applet_id: uuid.UUID,
+    user: User = Depends(get_current_user),
+    session=Depends(get_session),
+    query_params: QueryParams = Depends(parse_query_params(AppletTakeNowParams)),
+    answer_session=Depends(get_answer_session),
+) -> Response[TakeNowValidationResponse]:
+    is_valid = True
+    message = None
+    try:
+        await AppletService(session, user.id).exist_by_id(applet_id)
+        await CheckAccessService(session, user.id).check_answer_create_access(applet_id)
```

Comment:
Also validate that the current user is an admin with the appropriate roles:
- `super_admin`
- `owner`
- `manager`

Reply:
logged user should have permission to create a answer in applet, could be a normal user

Reply:
add validation returning 403 http status